lib/internal/process/per_thread.js

```diff
@@ -251,14 +251,23 @@ function wrapProcessMethods(binding) {
   /**
    * Loads the `.env` file to process.env.
    * @param {string | URL | Buffer | undefined} path
+   * @param {{
+   *    override?: boolean;
+   * }} [options]
    */
-  function loadEnvFile(path = undefined) { // Provide optional value so that `loadEnvFile.length` returns 0
+  function loadEnvFile(path = undefined, options = { override: false }) {
```

Comment:
Let's not create an object if we don't need to
```suggestion
  function loadEnvFile(path = undefined, options = kEmptyObject) {
```